### PR TITLE
Add guest auth to omniport proxy list

### PR DESCRIPTION
**Pull request by rish9101**

### CODE REVIEW

File: omniport/config/webpack.dev.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ module.exports = merge(common, {
         '/session_auth',
         '/open_auth',
         '/token_auth',
+	'/guest_auth',
```
Comment (algomaster99) [inline omniport/config/webpack.dev.js]: @rish9101 The indentation is a little off.

Comment (dhruvkb) [reply omniport/config/webpack.dev.js]: A little? It's not going work with that indentation afaik.

Comment (algomaster99) [reply omniport/config/webpack.dev.js]: @dhruvkb it's JS. Everything will work 😂

Comment (dhruvkb) [reply omniport/config/webpack.dev.js]: My bad. I confused that with Python 😂!

**Pull request merged**